### ISSUE TITLE
Quest bug: One Manlings`s Rubbish (Black Crag)

### ISSUE THREAD

**Issue by Kostoglod** [WarEmu/WarBugs#20373]
Good day! Quest bug: One Manlings`s Rubbish (Black Crag). The cave is blocked by barrier, i even did PQ here, its always blocked. No another entry.

**Comment by Kostoglod** [comment]
![Uploading diRruDv-R-4.jpg…]()
![8bPfRMik_g4](https://user-images.githubusercontent.com/111536343/186334443-0de25420-f443-4d18-9c86-ea42dda25c0a.jpg)

**Comment by Kostoglod** [comment]
![diRruDv-R-4](https://user-images.githubusercontent.com/111536343/186334509-ee2c612b-fd5b-4c81-9f08-a359b9af83ec.jpg)

**Comment by Rubyeclipse-war** [comment]
@Kostoglod Thanks for flagging this!

After investigating, it looks like the correct area to look is not Venom Cave, but the cave just up the hill from it that's taken over by Trolls. Within you'll find the "treasure" you seek! 

I've edited the Quest text to clarify this and give a better hint, and I've also moved the red marker that appears on your map to hint where to properly go. 

Finally, I've added some new completion text for both the first and second steps of this quest chain. This will go live with the next patch. Hope you enjoy, and happy questing!
